docs/PLUGIN_CATALOG.md

```diff
@@ -1,59 +1,64 @@
 This is the master catalog of plugins for snap. The plugins in this list may be written by multiple sources. Please examine the license and documentation of each plugin for more information.
 
 ## Maintained plugins
-
 | Name  | Type  | Description | Link | Download |
 | :---- | :---- | :---------- | :--- | :------- |
+| Apache | Collector | Collects metrics from the Apache Webserver for mod_status| [snap-plugin-collector-apache](https://github.com/intelsdi-x/snap-plugin-collector-apache) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-apache)
 | CEPH | Collector | Collects from CEPH cluster | [snap-plugin-collector-ceph](https://github.com/intelsdi-x/snap-plugin-collector-ceph) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-ceph)
+| Cinder | Collector | Collects from OpenStack Cinder | [snap-plugin-collector-cinder](https://github.com/intelsdi-x/snap-plugin-collector-cinder) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-cinder)
+| CPU | Collector | Collects CPU metrics from Linux procfs | [snap-plugin-collector-cpu](https://github.com/intelsdi-x/snap-plugin-collector-cpu) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-cpu)
+| DBI | Collector | Collects metrics from SQL DBs using go's "sql" package | [snap-plugin-collector-dbi](https://github.com/intelsdi-x/snap-plugin-collector-dbi) | [Linux](https://s3-us-west-1.amazonaws.com/snap-plugins-linux-latest/snap-plugin-collector-dbi)
```

Comment:
Consider changing the description of snap-plugin-collector-dbi to something like this: "Collects metrics as a results of executing SQL statements on database (MySQL and PostgreSQL supported)"